### PR TITLE
try and install master sunpy on travis

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,13 +33,13 @@ env:
...
         - SETUP_CMD='test'
-        - PIP_DEPENDENCIES='git+https://github.com/sunpy/cube.git'
+        - PIP_DEPENDENCIES='git+https://github.com/sunpy/sunpy.git git+https://github.com/sunpy/cube.git'
         - EVENT_TYPE='pull_request push'
 
         # For this package-template, we include examples of Cython modules,
         # so Cython is required for testing. If your package does not include
         # Cython code, you can set CONDA_DEPENDENCIES=''
-        - CONDA_DEPENDENCIES='sunpy xarray pytest coverage sphinx sphinx_rtd_theme'
+        - CONDA_DEPENDENCIES='astropy scipy pandas matplotlib suds-jurko glymur sqlalchemy beautifulsoup4 requests scikit-image xarray pytest coverage sphinx sphinx_rtd_theme'
```
Comment (DanRyanIrish) [inline .travis.yml]: Can remove xarray from this list of dependencies.  We no longer use it.

**Comment by DanRyanIrish** [comment]
Didn't work...Hmmm.  I wonder what is the answer?

**Comment by Cadair** [comment]
ah that looks like a packaging bug in cube. I will have to investigate when I have more time.

**Comment by DanRyanIrish** [comment]
At least you have an idea of what it might be now :)

**Pull request closed without merging**